tests/FakeItEasy.Analyzer.Tests/UnusedReturnValueAnalyzerTests.cs

```diff
@@ -110,6 +110,39 @@ interface IFoo { int Bar(); }
         [Test]
         [SetUICulture("en-US")] // so that the message is in the expected language regardless of the OS language
         [SuppressMessage("Microsoft.Naming", "CA2204:Literals should be spelled correctly", MessageId = "CallTo", Justification = "It's an identifier")]
+        public void Diagnostic_Should_Be_Triggered_When_Call_Specification_Is_Not_Used_With_UsingStatic()

```

Comment:
:+1: @blairconrad 

I think I'll pick your first suggestion for consistency with the other tests